src/less/components/payex/form.less

```diff
@@ -3,3 +3,53 @@
 .help-block {
 	color: @help-block-text;
 }
+
+
+.form-group {
+	&.disabled,
+	[disabled] {
+		color: @brand-secondary-light-3;
```

Comment:
Only variable not using the css vars (--). Might update so we're using same approach everywhere